pages/History.qml

```diff
@@ -170,6 +170,21 @@ Rectangle {
                         root.updateFilter();
                     }
                 }
+
+                MoneroComponents.InlineButton {
+                    Layout.topMargin: -8
+                    Layout.rightMargin: -8
+                    Layout.leftMargin: 8
```

Comment:
```suggestion
                    Layout.leftMargin: -2
```

Reply:
I think it's necessary enough blank space after the last character, so the user can click at the end of the search bar content with the mouse. By using a small left margin it's easy to accidentally click the clean search button.

Reply:
But even if you click on the right side it does not put you to the end of the text.

Reply:
ok, no problem